### PR TITLE
Search: edit 1 / 3 / 10 year vision

### PR DESCRIPTION
**Pull request by lguychard**
This PR proposes an edit to the 1 / 3 / 10 year vision statements on our top-level [Search strategy](https://handbook.sourcegraph.com/company/strategy/code-graph/search) page, to make them less hand-wavey, and more grounded in our org and company strategies.

- 1-year: a continuity of what Search Core and Search Product are currently working on.
- 3-year: ties into the Code Graph vision for a unified feature set, and our OSS scaling ambitions.
- 10-year: ties into the top-level company vision for coding literacy.

A follow-up here could be to add examples of each vision statement being realized.

@sourcegraph/search-product, @sourcegraph/search-core would love to get feedback here -- does this resonate?

### CODE REVIEW

File: content/company/strategy/code-graph/search/index.md [hunk 1]
```diff
@@ -15,29 +15,28 @@ Sourcegraph's mission is to make it so that everyone can code. The Search teams'
 
 ## Vision
 
-Sourcegraph search is a critical part of development workflows because it provides the most precise, relevant search results and empowers users to code with confidence.
-
 #### 1 Year vision
 
-> Developers can find all the code they need and code search is a critical part of their workflow, increasing individual and team productivity.
+> Enterprise customers of Sourcegraph actively use our differentiating code search features. Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results.
```
Comment (benvenker) [inline content/company/strategy/code-graph/search/index.md]: maybe "differentiated"?

**Review by benvenker — COMMENTED** [review]
I like these. I also had some updates on a local branch but these are more succinct and get the same message across!

**Review by camdencheek — APPROVED** [review]

File: content/company/strategy/code-graph/search/index.md (hunk 1 above)
Comment (lguychard) [reply content/company/strategy/code-graph/search/index.md]:
```suggestion
> Enterprise customers of Sourcegraph actively use our differentiated code search features. Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results.
```

Yes, good point

**Review by limitedmage — APPROVED** [review]

**Comment by lguychard** [comment]
Merging since this has a few approvals, happy to address feedback post-merge or as a PR :)

**Pull request merged**